documentation/publish/distribute-sideloading.md

```diff
@@ -0,0 +1,256 @@
+
+   ![Add-ons Manager setting cog]({% asset "documentation/publish/install-addon-from-file.png" @optim @path %})
+
+4. Browse to and open the file from the location where it was saved.
+
+5. When prompted click **Add**:
+
+   ![add add-on confirmation]({% asset "documentation/publish/add_add_on_confirmation.png" @optim @path %})
+
+6. The add-on will now appear in the add-on manager’s list of installed add-ons and be ready to use:
+
+   ![add-on appears in add-ons manager]({% asset "documentation/publish/add_on_added.png" @optim @path %})
+
+{% endcapture %}
+{% include modules/one-column.html
+  id="install-addon-from-file"
+  content=content
+  aside=""
+%}
+
+<!-- END: Single Column Body Module -->
+
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## Installation using the standard extension folders
+
+This method of add-on installation involves copying the add-on into one of the standard extension folders on the user's computer. Once copied, the next time Firefox launches the add-on will be installed. By default, the user will be asked to approve the installation, and if the user approves, the add-on will be installed and automatically loaded for subsequent launches. If the user has more than one Firefox profile, the approval and installation will occur on the next launch of each profile. For details on controlling whether the user is prompted to approve the installation, see [Controlling automatic installation](/documentation/enterprise/enterprise-distribution/#firefox-settings).
+
+### Rename your XPI file
+
+To use this method, your XPI file must be named using the add-on or application ID, as set in [Preparing your add-on](#preparing-your-addon). The signed add-on file you downloaded from AMO will be named something like borderify-1.0-an+fx.xpi (see [Signing and distributing your add-on](https://developer.mozilla.org/docs/Mozilla/Add-ons/Distribution) for more details), change this to, for example, borderify@example.com.xpi.
+
+<!-- Note -->
+
+{% capture note %}
+
+If you are developing an add-on for Firefox you can use an extension proxy file to install an add-on without copying the files over to the standard extensions folders.
+
+{% endcapture %}
+{% include modules/note.html
+	content=note
+	alert=false
+%}
+
+<!-- END: Note -->
+
+### Add the add-on XPI file to a standard extensions folder
+
+In what follows {ec8030f7-c20a-464f-9b0e-13a3a9e97384} is the application ID of Firefox.
+
+The standard installation of Firefox disables the automatic installation of add-ons from these locations (see [Controlling automatic installation](/documentation/enterprise/enterprise-distribution/#firefox-settings)). As a result, the process for each of the methods described below is as follows:
+
+- Copy the renamed XPI file to the extensions folder for Windows, OSX, or Linux as appropriate. Note that, depending on the desktop OS and its settings, the user may need administrator permission to complete this action.
```

Comment:
Should "Windows", "OSX", and "Linux" be linked to the sections below (like on MDN?)